Bluejay/Bluejay/Bluejay.swift

```diff
@@ -558,6 +563,9 @@ extension Bluejay: CBCentralManagerDelegate {
         UIApplication.shared.endBackgroundTask(backgroundTask)
     }
     
+    /**
+     Examine the listen cache in `UserDefaults` to determine whether there are any listens that might need restoration.
```

Comment:
IMO if we want to explain in a comment how a private method works internally, that should go into inline comments in the method rather than in a separate doc comment that is liable to get out of date.